cmd/populator/docs/populator_register-ocf-manifests.md

```diff
@@ -28,13 +32,10 @@ kubectl -n capact-system port-forward svc/neo4j-neo4j 7687:7687
 kubectl -n capact-system port-forward svc/neo4j-neo4j 7474:7474
 ```
 
-
-It will create a `populator` binary in a local dir.
-
-To run it and use local manifests from Capact repo:
+To run it and use manifests for example from [hub-manifests](https://github.com/capactio/hub-manifests) repo:
 
 ```shell
-./populator register ocf-manifests .
+./bin/populator-linux-amd64 register ocf-manifests <path to the main directory of the repo>
```

Comment:
```suggestion
./bin/populator-linux-amd64 register ocf-manifests {PATH_TO_THE_MAIN_DIRECTORY_OF_THE_REPO}
```

to be consistent with the same definition as we have below for the `{HOST}` arg.